test/storage/index.js

```diff
@@ -0,0 +1,116 @@
+const { config, run } = require('hardhat');
```

Comment:
now what i'd suggest is creating another file here sitting next to index.js to follow the pattern for tests so far.

I know it's a single file but in case we break things down and to keep it consistent we want the index.js file to be just that, the index of all the tests such as here:

https://github.com/NexusMutual/smart-contracts/blob/862539ae93e7c00d5ccb9069f5afa0c5fcd4704e/test/unit/Assessment/index.js#L4